tests/cupyx_tests/scipy_tests/sparse_tests/test_coo.py

```diff
@@ -861,7 +861,10 @@ def test_rmul_dense_ndim3(self):
             with pytest.raises(ValueError):
                 x * m
 
-    @testing.with_requires('scipy!=1.8.0rc1')  # scipy/15210
+    @testing.xfail(
+        numpy.lib.NumpyVersion(scipy.__version__) >= '1.8.0rc1',
+        reason='See scipy/15210')
+    @testing.with_requires('scipy>=1.8.0rc1')
```

Comment:
I think all these `@testing.with_requires('scipy>=1.8.0rc1')` can be removed?

Reply:
Yes, I mixed them up with SciPy 1.8's other new behavior...